bin/index.js

```diff
@@ -19,6 +19,7 @@ const params = program
     .option('--exportCore <value>', 'Write core files to disk', true)
     .option('--exportServices <value>', 'Write services to disk', true)
     .option('--exportModels <value>', 'Write models to disk', true)
+    .option('--ignoreOperationId <value>', 'Use operation id', true)
```

Comment:
```suggestion
    .option('--ignoreOperationId <value>', 'Ignore operation id', true)
```

Although I agree with the initial intent – make the flag opt-in and default to true to preserve current functionality

```suggestion
    .option('--useOperationId <value>', 'Use operation id', true)
```

Reply:
Fixed in the next commit. Thanks for the review.